defaults/main.yml

```diff
@@ -599,3 +599,13 @@ nft_backup_conf: True
                                                                    # ]]]
                                                                    # ]]]
                                                                    # ]]]
+# OS specific variables defaults [[[
+# ----------------------------------
+
+# .. envvar:: nft_bin_location [[[
+#
+# Specify Nftables executable location.
+#
+nft_bin_location: '/usr/sbin/nft'
```

Comment:
> > Also, can you add a double underscores in the new variable, right after role name (eg. nft__bin_location instead of nft_bin_location) ?
> 
> Done. I decided to keep it in the same commit, let me know if it should be split.

The comment of your commit seems good but it's still nft_bin_location in the code. You can add a new commit if you prefer ;)

Reply:
Looks like an operator error on my behalf - did not actually add the changes into amended commit :)